libs/net/dal/Services/TonePoolService.cs

```diff
@@ -23,5 +24,19 @@ public IEnumerable<TonePool> FindAll()
             .AsNoTracking()
             .OrderBy(a => a.SortOrder).ThenBy(a => a.Name).ToArray();
     }
+
+    public TonePool? FindByUserId(int userId)
+    {
+        return this.Context.TonePools.FirstOrDefault(tp => tp.OwnerId == userId && tp.IsPublic == true);
```

Comment:
We're presently only going to create one tonepool for each user.  It doesn't matter if it's public.